modules/dcache-chimera/src/main/java/org/dcache/chimera/namespace/ChimeraCleaner.java

```diff
@@ -75,6 +75,9 @@
     private static final Logger _log =
         LoggerFactory.getLogger(ChimeraCleaner.class);
 
+    // ToDo: Spring Implementation for queryLimit ( used in runNotification() )
```

Comment:
It's considered an annotation for future reference as I talked with @kofemann about the proper implementation of the limit. Idealy it would be in a properties file, but that seems more difficult to implement for now and we agreed on defining the QueryLimit as a constant variable in the class.